### PR TITLE
upgraded to Play 2.7.0-RC3

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.3"))
+addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.4"))
```
Comment (ignasi35) [inline project/plugins.sbt]: Is this change still necessary if we remove the build for `scala213`? (I guess it doesn't harm, correct?)

File: README.md [hunk 2]
```diff
@@ -22,4 +22,4 @@ The Play Ebean plugin supports several different versions of Play and Ebean.
 | 2.0.0          | 2.4.x        | 6.8.x         |
 | 1.0.0          | 2.4.x        | 4.6.x         |
 
-> * Milestone releases: these releases are not stable and should not be production.
+> * Release Candidate: these releases are not stable and should not used in be production.
```
Comment (ignasi35) [inline README.md]:
```suggestion
> * Release Candidate: these releases are not stable and should not be used in production.
```

File: project/plugins.sbt (hunk 1 above)
Comment (renatocaval) [reply project/plugins.sbt]: no, this is not necessary. I did it only for the sake of maintenance
